man/systemd-run.xml

```diff
@@ -394,6 +394,28 @@ Dec 08 20:44:48 container systemd[1]: Started /bin/touch /tmp/foo.</programlisti
 
       <programlisting># systemd-run -t --send-sighup /bin/bash</programlisting>
     </example>
+
+    <example>
+      <title>Start <command>screen</command> as a user service</title>
+
+      <programlisting>$ systemd-run --scope --user screen -d -m
+Running scope as unit run-r14b0047ab6df45bfb45e7786cc839e76.scope.
+
+$ screen -ls
+There is a screen on:
+        492..laptop     (Detached)
+1 Socket in /var/run/screen/S-fatima.
+</programlisting>
+
+      <para>To allow the screen to persist after the user logs out,
+      even if the session scope is terminated, enable lingering:</para>
+
```

Comment:
Yes, I misunderstood what linger does and your new version is better. But even that is not enough; e.g. when running something like `systemd-run --scope --user /some/program &` this program will be sent SIGHUP by shell when exiting, one still needs nohup to prevent it. May be systemd-run should do it itself; but until it does man page better mentions it.